### ISSUE TITLE
Cannot get to the URL or ID of a Flow Webhook Trigger

### ISSUE THREAD

**Issue by st11x** [directus/directus#21616]
### Describe the Bug

When I create a webhook Flow trigger, the URL to that trigger is shown with ellipsis but there is nowhere to copy the full URL.

![image](https://github.com/directus/directus/assets/7398499/b66c37af-59b7-4667-b576-1d75f9408f32)

The documentation talks about a webhook trigger panel and I suppose that is it.

![image](https://github.com/directus/directus/assets/7398499/8a653d9d-3cc8-4205-98a7-1eda269a0baa)

Workaround: I could use the browser inspect tool to get the URL. And since the ID is in the browser, I can deduce the URL to the webhook.

### To Reproduce

Create a Flows Webhook trigger

### Directus Version

10.9.3

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by azrikahar** [comment]
That's odd indeed. The copy button should appear on the right of the URL, as seen here:

![image](https://github.com/directus/directus/assets/42867097/23649d58-819d-406b-8f96-784e9e45659e)

as well as the screenshot in the docs.

That being said, the button does check whether your browser currently supports copying to clipboard (or has the permission to do so) as seen with the `isCopySupported`:

https://github.com/directus/directus/blob/79c2701a74919cff3c412be8caa4c900f1f2aed2/app/src/modules/settings/routes/flows/components/options-overview.vue#L29-L36

https://github.com/directus/directus/blob/79c2701a74919cff3c412be8caa4c900f1f2aed2/app/src/composables/use-clipboard.ts#L13-L15

Can you check whether copying to clipboard is allowed in your current browser? or perhaps checking with another browser as well?

Closing this for now as I believe it's a browser permission issue, but if it still persists with the necessary permission, let's reopen this and investigate further.

**Comment by st11x** [comment]
I have checked Edge/Chrome on Win 11, both regular and incognito. I have also checked that on Safari on the iPad. All of them are missing that icon.

Copy permission is good. I can select and copy all the visible parts of the URL minus the ellipsis.

**Comment by paescuj** [comment]
Do you access the page over HTTP and non-localhost domain?
The copy action is only allowed over HTTPS or via HTTP when the domain is 127.0.0.1/localhost.

**Comment by st11x** [comment]
Yes it is over http. I had suspected that originally but wasn't sure. Thanks.

**Comment by paescuj** [comment]
That being said, I think we should provide an alternative way for users to see the full URL. At least a tooltip.
Therefore, I'm reopening this issue so we can address this as a small UI improvement 👍

**Comment by FutureBuddha** [comment]
i use brave browser, directus 10.10.7 and the same situation 
![image](https://github.com/directus/directus/assets/149382049/5d4a951a-0f51-4102-9ae3-b89494885e97)
is there already something done to get the url or the id, except out of the url to get the id?
